scripts/common/tasks-helpers.ts

```diff
@@ -0,0 +1,49 @@
+import { HardhatRuntimeEnvironment } from 'hardhat/types'
+import { HardhatUtils } from './hardhat.utils'
+import { BigNumber } from 'bignumber.js'
+import { AutomationBot } from '../../typechain'
+import { getStartBlocksFor } from './addresses'
+import { bignumberToTopic } from './utils'
+import { Network } from './types'
+
+export async function getProxy(hre: HardhatRuntimeEnvironment, hardhatUtils: HardhatUtils, vault: string) {
+    let isMaker = false
```

Comment:
you can just do `parseInt(vauit,10)` if it's not 0 its Maker

Reply:
but honestly I think that should be yet another flag, like --usedpm  determining how we access position in principle we can have dpm Maker positions.

Reply:
yeah good idea. I'll do it this way